acceptance/cmd/workspace/apps/run-local/script

```diff
@@ -0,0 +1,34 @@
+cd app
+
+trace errcode $CLI apps run-local --entry-point value-from.yml 2>&1
+
+# We first run the command with different entry point which starts unblocking script
+# so we don't need to start it in background. It will install the dependencies as part of the command
+trace $CLI apps run-local --prepare-environment --entry-point test.yml 2>&1 | grep -w "Hello, world"
+
+title "Starting the app in background..."
+$CLI apps run-local --prepare-environment --debug > ../out.run.txt 2>&1 &
+PID=$!
+# Ensure background process is killed on script exit
+trap 'kill $PID 2>/dev/null || true' EXIT
+cd ..
+
+title Waiting for the app to start...
+# Use a loop to check for the startup message instead of tail/sed which can be unreliable on Windows
+# due to file locking, buffering issues, and different text processing behavior across Windows versions.
+# A simple grep loop is more robust across platforms.
+while [ -z "$(grep -o "Python Flask app has started with" out.run.txt 2>/dev/null)" ]; do
+    sleep 1
```

Comment:
Can we make this more resilient? test a few times instead of forever? or have some other signal?

I'm running `go test . -run ^TestAccept$ -update` and I have this test hanging forever.

<img width="1300" alt="Screenshot 2025-04-30 at 17 43 52" src="https://github.com/user-attachments/assets/8722f235-b01a-4eb3-9c2a-1f360520d50c" />

Reply:
Fixed here https://github.com/databricks/cli/pull/2794